samples/copyclean.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#    This file is part of python-registry.
+#    It tries to make a clean copy of hive file
+#    clean meaning without unused (not understood) data
+#
+#   Copyright 2015 Christian Nilsson <nikize@gmail.com>
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from __future__ import print_function
+
+import sys
+
+from Registry import RegistryParse
+import struct
+
+class Copy(RegistryParse.REGFBlock):
+    """
+    Parsing of file as a Windows Registry file.
+    """
+    def __init__(self, filelikeobject, writer):
+        """
+        Constructor.
+        Arguments:
+        - `filelikeobject`: A file-like object with a .read() method.
+              If a Python string is passed, it is interpreted as a filename,
+              and the corresponding file is opened.
+        """
+        try:
+            self._buf = filelikeobject.read()
+        except AttributeError:
+            with open(filelikeobject, "rb") as f:
+                self._buf = f.read()
+        self._buflen = len(self._buf)
+        
+        self._writer = writer
+
+        super(Copy, self).__init__(self._buf, 0, False)
+
+    def pack_dword(self, offset, *data):
+        """
+        write little-endian DWORDs (4 bytes) to the relative offset.
+        Arguments:
+        - `offset`: The relative offset from the start of the block.
+        - `data`: The data to be writen, can be multiple
+        """
```

Comment:
The credit should probably go back to you since it's mostly a copy of the existing functions.
Now when I think about how much was copied - i will add you as copyright header as well.